package.json

```diff
@@ -68,13 +68,16 @@
     "lz-string": "^1.4.4",
     "pretty-format": "^27.0.2"
   },
+  "overrides": {
+    "jsdom": "^21.1.0"
+  },
   "devDependencies": {
     "@testing-library/jest-dom": "^5.11.6",
     "@types/lz-string": "^1.3.34",
     "jest-in-case": "^1.0.2",
     "jest-snapshot-serializer-ansi": "^1.0.0",
     "jest-watch-select-projects": "^2.0.0",
-    "jsdom": "^16.4.0",
+    "jsdom": "^21.1.0",
```

Comment:
quite the hefty bump pulling in unrelated breaking changes. Will bump kcd-scripts to jest 29 first, then update kcd-scripts here